src/chapters/strings/template-literals.rst

```diff
@@ -0,0 +1,59 @@
+Template Literals
+=================
+
+.. index:: ! template literal
+
+The ECMAScript specifications define the standard for JavaScript. The 6th edition, known as ES2015, added **template literals**, which allow for the automatic insertion of expressions (including variables) into strings.
```

Comment:
Rephrasing suggestion: "**Template literals** allow for the automatic insertion of expressions (including variables) into strings."

Reply:
Is your suggestion to break this paragraph into 3 sentences, or to remove the first portion of the second sentence?

Reply:
My thought was to start with "Template literals" and remove the preceding text.

Reply:
Sounds good. I'll put the comment about ES2015 in a Note, since, while not relevant to the definition, students should know that it's a relatively new (and not universally-supported) feature.